finetune/lora.py

```diff
@@ -93,10 +93,10 @@ def setup(
     logger = CSVLogger(out_dir.parent, out_dir.name, flush_logs_every_n_steps=log_interval)
     fabric = L.Fabric(devices=devices, strategy=strategy, precision=precision, loggers=logger, plugins=plugins)
     fabric.print(hparams)
-    fabric.launch(main, data_dir, checkpoint_dir, out_dir, quantize)
```

Comment:
This is the leftover from [previous](https://github.com/Lightning-AI/lit-gpt/blob/cc190152085ac60fe708455b2df8877f82d37087/finetune/lora.py#L88) quantization approach.